src/wizard/wizard-directive.js

```diff
@@ -0,0 +1,677 @@
+/**
+  * @ngdoc directive
+  * @name patternfly.wizard.directive:pfWizard
+  *
+  * @description
+  * Directive for rendering a Wizard modal.
+  *
```

Comment:
I think more information is needed here. The overall structure of the wizard directive, step directive, and sub step directives.